python/TestHarness/util.py

```diff
@@ -528,7 +528,12 @@ def getCapabilities(exe):
     Get capabilities JSON and compare it to the required capabilities
     """
     assert exe
-    output = runCommand("%s --show-capabilities" % exe)
+    try:
+        output = runCommand("%s --show-capabilities" % exe)
+    except:
+        print("Error! Executable crashed.")
+        exit(1)
+
```

Comment:
You could put this `try`/`catch` into `runCommand`...

Reply:
It will change what people see, instead of seeing a bunch of test failed because it crashed, they will only see:

Error! Executable crashed.

Reply:
I was wrong, I wasn't catching the exception.  I pasted in the error messages below.  I added some extra output to the exception that has always been raised.